tests/test_detection_mode.py

```
@@ -0,0 +1,99 @@
+        assert mode == "documentation"
+        assert conf > 0.0
+
+    def test_documentation_explain(self) -> None:
+        mode, conf = classify_mode(
+            "explain how the authentication module works"
+        )
+        assert mode == "documentation"
+        assert conf > 0.0
+
+    def test_chat_default(self) -> None:
+        mode, conf = classify_mode("hello, how are you?")
```

Ruff (0.15.9)

[warning] Unpacked variable `conf` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)